source/dmocks/action.d

```diff
@@ -64,8 +64,8 @@ struct Actor
             {
                 import std.format : format;
 
-                throw new Error(format!"cannot call action: type %s does not match argument type %s"
-                    (self.action.typename, ArgTypes.stringof));
+                throw new Error(format!"cannot call action: type %s does not match argument type %s"(
```

Comment:
This is not exactly "keep line breaks": bug or feature?

Reply:
Kind of feature, or laziness. dfmt seems never to break between the template and functions arguments, therefore it can't indent the function argument list on its own properly.
If I keep the newline here, I get:

```diff
 void f()
 {
     format!"cannot call action: type %s does not match argument type %s"
-        (self.action.typename, ArgTypes.stringof);
+    (self.action.typename, ArgTypes.stringof);
 }
```

So in this case proper indentation handling should be added as well.

Reply:
... and it makes unclear how to indent arguments

```d
format!"cannot call action: type %s does not match argument type %s"
        (
                self.action.typename,
                ArgTypes.stringof
        );
 }
```
or
```d
format!"cannot call action: type %s does not match argument type %s"
        (self.action.typename,
        ArgTypes.stringof);
 }
```